src/clojure/abracad/avro/write.clj

```diff
@@ -207,13 +208,17 @@ record serialization."
   (field-get [this field] (get this field))
   (field-list [this] (keys this)))
 
-(defn avro-record?
+(defn avro-record-score
```

Comment:
Hmm, now I look again, I'm not so sure about this. An implication of this scoring mechanism is that a record type will *always* be selected regardless of whether it is actually suitable. A score should only be generate if the match is suitable.